i-RIC/iricdev-2019
# Issue 1

To test please do the following:

1. Rename `lib` directory to `lib.save`
2. Edit `msbuild_2019.cmd` so that 
`msbuild -noLogo -maxCpuCount -target:all iricdev.proj`
is replaced with
`msbuild -noLogo -maxCpuCount -target:geos-build iricdev.proj`
3. In a command prompt run `msbuild_2019.cmd`
4. Verify that both `lib\install\geos-3.8.1\[debug,release]\lib\geos.lib` are created
5. Delete new `lib` directory
6. Rename `lib.save` directory to `lib`
7. Update msbuild_2019.cmd
`git restore msbuild_2019.cmd`

Hi Keiksuke,
There was a bug in the build directories (both were built in the same directory.  So, after verifying the build, you might want to delete the original `lib\[src,build,install]\geos-3.8.1` and rerun `msbuild_2019.cmd`
